src/plugins/saved_objects/README.md

```diff
@@ -175,3 +175,656 @@ The migraton version will be saved as a `migrationVersion` attribute in the save
+        },
+        {
+            "type": "index-pattern",
+            "id": "619cc200-ecd0-11ee-95b1-e7363f9e289d",
+            "meta": {
+                "title": "test*",
+                "icon": "indexPatternApp"
+            },
+            "destinationId": "ffd3719c-2314-4022-befc-7d3007225952"
+        },
+        {
+            "type": "index-pattern",
+            "id": "test-pattern1",
+            "meta": {
+                "title": "test-pattern1-*",
+                "icon": "indexPatternApp"
+            },
+            "destinationId": "e87e7f2d-8498-4e44-8d25-f7d41f3b3844"
+        }
+    ]
+}
+```
+
+### Resolve import saved objects errors API
+
+Resolve the errors if the import API returns errors, this API can be used to retry importing some saved obejcts,  overwrite specific saved objects, or change the references to different saved objects.
+
+* Path and HTTP methods
+
+```json
+POST <osd host>:<port>/api/saved_objects/_resolve_import_errors
+```
+
+* Query parameters
+
+| Parameter | Data type | Required | Description |
+| :--- | :--- | :--- | :--- |
+| `createNewCopies` | Boolean | NO | Creates copies of the saved objects, genereate new IDs for the imported saved obejcts and resets the reference. |
+| `dataSourceId` | String | NO  | The ID of the data source. |
+| `workspaces` | String|Array | NO  | The ID of the workspace which the saved objects exist in. |
+
+* Request body
+
+The request body must include a multipart/form-data.
+
+| Parameter | Data type | Required | Description |
+| :--- | :--- | :--- | :--- |
+| `file` | ndjson file | YES | The same file given to the import API. |
+| `retries` | Array | YES | The retry operations. |
+
+The attrbutes of the object in the `objects` parameter are as follows:
+| Parameter | Data type | Required | Description |
+| :--- | :--- | :--- | :--- |
+| `type` | String | YES | The type of the saved object, such as `index-pattern`, `config` and `dashboard`. |
+| `id` | String | YES |The ID of the saved object. |
+| `overwrite` | Boolean | NO | If `true`, overwrite the saved object with the same ID, defaults to `false`. |
+| `destinationId` | String | NO | The destination ID that the imported object should have, if different from the current ID. |
+| `replaceReferences` | Array | NO | A list of `type`, `from`, and `to` to be used to change the saved object's references. |
+| `ignoreMissingReferences` | Boolean | NO | If `true`, ignores missing reference errors, defaults to `false`. |
+
+* Example request
+
+```json
+POST api/saved_objects/_import?createNewCopies=true --form file=@export.ndjson --form retries='[{"type":"index-pattern","id":"my-pattern","overwrite":true}]'
+
+```
+
+* Example response
+
+```json
+{
+    "successCount": 0,
+    "success": true
+}
+```
+
+### Duplicate saved objects API
+
+Duplicate saved objects among workspaces.
+
+* Path and HTTP methods
+
+```json
+POST <osd host>:<port>/api/workspaces/_duplicate_saved_objects
```

Comment:
This is more related to workspace. Should we add this in workspace docs?

Reply:
Sure, I've moved this API's doc to the workspace plugin's doc, even though currently we don't have a workspace doc, but same to this doc file, I've created a new markdown file in workspace plugin and added the docs of all the server APIs in workspace plugin to the file.